deploy/docker/Dockerfile

```diff
@@ -1,28 +1,44 @@
 FROM alpine:3.5
 
-LABEL maintainer="adundovi@gmail.com"
-LABEL description="Prepares a docker image \
-of pump.io server, http://pump.io/."
+LABEL maintainer Andrej Dundovic <adundovi@gmail.com>
+LABEL description "Stream server that does \
+most of what people really want from a social network."
 
 ENV LC_ALL en_US.UTF-8
 
+ENV PUMPIO_VERSION 3.0.2
+
 RUN apk add --no-cache \
-  nodejs \
-  git \
-  python \
-  make \
-  g++ \
-  graphicsmagick
-RUN npm install -g databank-mongodb
-RUN npm install -g pump.io
-RUN adduser -D -g "" pumpio
+    curl \
+    nodejs \
+    python \
+    make \
+    g++ \
+    git \
+    graphicsmagick \
+ && adduser -D -g "" pumpio \
+ && mkdir -p /etc/ssl/private \
+ && npm install -g databank-mongodb@0.19.2 \
+ && mkdir -p /usr/src \
+ && curl -fSL https://github.com/pump-io/pump.io/archive/v$PUMPIO_VERSION.zip -o pumpio.zip \
```

Comment:
I have read your links, and I now understand this use case, but I must say it is a totally different approach compared to a deployment container because a source code is started from the outside of the container (that's why there are no rebuilds). Therefore, I recommend that we split the docker story to deployment and development one. For the former one we start a new repository which would be updated only when needed (ideally, rarely) and the later which would follow the practice from the first link you submitted and where the dockerfile would be in the root directory of this repository.

For the deployment one - I don't want to manually checkout a github repository, maybe I don't even have git installed on my host. I just want to have a pump.io instance running by using docker pull command.